index.md

```diff
@@ -110,28 +110,23 @@ While not fixed requirements these are a list of things to consider when choosin
 A set of automated systems will be employed to verify that all vendored modules
 continue to meet the policy.
 
-* `mach lint node`: Primarily intended for CI, runs checks that no modules or
-dependencies use a disallowed license or have known security vulnerabilities.
-Verifies that `package-lock.json` is valid (does not pull code from unexpected
-registries).
+* `mach lint node`: Runs checks that no modules or dependencies use a disallowed license
+or have known security vulnerabilities.  Verifies that `package-lock.json` is valid (does
+not pull code from unexpected registries).  This is intended to be used locally (by
+`mach vendor node`), as well periodically on CI (to maintain the invariants).
```

Comment:
Is `mach vendor node` also going to be used to update packages? I guess that would make sense.

Reply:
That's how I've been thinking about it, yeah.